ecommerce-ui/src/cartTotal/cartTotal.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import CartTotal from './cartTotal';
+import TestRenderer from 'react-test-renderer'
+
+describe("AddRental", () => {
+    it('Smoke test make sure the component rendors', () =>{
+        const div = document.createElement('div');
+        ReactDOM.render(<CartTotal/>, div);
```

Comment:
Nice job with your tests. These two are failing though. You'll want to ensure that you pass in some "test" props to ensure the CartTotal component renders appropriately